package.json

```diff
@@ -38,10 +38,11 @@
     "classnames": "^2.2.6",
```

Comment:
Why do we need `classnames` in the peer dependencies?

Reply:
I guess to avoid multiple versions of the lib